core/database_util.go

```diff
@@ -234,6 +238,67 @@ func GetBlock(db ethdb.Database, hash common.Hash) *types.Block {
 	return types.NewBlockWithHeader(header).WithBody(body.Transactions, body.Uncles)
 }
 
+// GetBlockReceipts retrieves the receipts generated by the transactions included
+// in a block given by its hash.
+func GetBlockReceipts(db ethdb.Database, hash common.Hash) types.Receipts {
+	data, _ := db.Get(append(blockReceiptsPrefix, hash[:]...))
```

Comment:
Hmm, by `data` I really meant `binary blob`, just didn't want to write that all out. A better option would be plain `rlp`, but that would conflict with the package. Hence why I left it with just `data`, didn't want to write a lot of different variable names that just mean `binary blob`.